pkg/server/server.go

```diff
@@ -87,10 +88,30 @@ func (server *Server) Run(group task.Group) error {
 		return fmt.Errorf("Server.Run: endpoint creation failed with %s", err)
 	}
 
+	// Enable CORS
+	cors := cors.New(cors.Options{
+		AllowedOrigins: []string{"http://localhost:3000", "https://juiceweb.vercel.app"},
```

Comment:
Can we pass these in as arguments to the application? I am trying to keep this more generic for onprem usecases but I also understand for development hardcoding is easier.

Reply:
Do you think we'll host a separate instance of juiceweb for onprem controllers? I was leaning towards actually just disabling CORS altogether for that.

Reply:
I don't know how juiceweb would work for onprem. Are you thinking they would launch their own juiceweb deployment as well as the controller?

Reply:
i think it'll really depend on the requirements for these? I imagine most wont need any kind of juiceweb UI at all since they'll do login independently. If they do need some kind of API host, I think we'll split the API portion out (which is tiny anyway)

Reply:
The CORS headers are only read by browsers so really only relevant in the publicly-hosted controller scenario.